tests/test_venue_request.py

```diff
@@ -281,7 +281,7 @@ def test_venue_recruitment(self, client, test_client, selenium, request_page, ve
         assert reply_row
         buttons = reply_row.find_elements_by_class_name('btn-xs')
         assert [btn for btn in buttons if btn.text == 'Recruitment']
-
+        #assert venue.recruitment_content.invitee_details['description'] == 'Enter a list of invitees with one per line. Either tilde IDs or email,name pairs expected. Eg E.g. captain_rogers@marvel.com, Captain America or ~Captain_America1' #added by me, does not work 
```

Comment:
why is this line commented?

Reply:
if you want to check that the description looks good, you need to get the recruitment invitation for this venue (`invitation='{}/-/Request{}/Recruitment'.format(venue['support_group_id'], venue['request_form_note'].number)`) and then check the description  for the 'invitee_details' field in the reply.content of that invitation

Reply:
Oh I see you did this further down! no need for this line then